xmake/toolchains/dpcpp/xmake.lua

```diff
@@ -0,0 +1,35 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- Copyright (C) 2015-present, TBOOX Open Source Group.
+--
+-- @author      ruki
+-- @file        xmake.lua
+--
+
+-- define toolchain
+toolchain("dpcpp")
+
+    -- set homepage
+    set_homepage("https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compiler.html")
+    set_description("Intel LLVM C/C++ Compiler")
```

Comment:
It is also the `Intel LLVM C/C++ Compiler` ? What is the difference between it and icx?

Reply:
Could you use a different description to distinguish between icx and dpcc?

Reply:
yes, it's another intel compiler that uses SYCL.

But I just realized that it only allows to compile C sources. Is removing `toolchain:set("toolset", "cc", "dpcpp")` enough ? Should I remove `set_kind("standalone")` ?

Reply:
need not remove it